tests/test_utils.py

```diff
@@ -233,6 +235,10 @@ def test_bom_generation_with_default_variant(request, instance):
         "Archimajor.PrjPcb",
         attributes_mapping,
         ref="916e739f3ad9d956f4e2a293542050e1df9e6f9e",
+        # For the variants tests, we don't want to remove non-BOM components
+        # because some of them are enabled by the variants, and we want to
+        # test that they are included when required.
+        remove_non_bom_components=False,
```

Comment:
Would users using variants need to think about this? Or is it just us for testing?

Reply:
Just for us testing, so you can visually confirm that the correct number of components are removed when you compare with the project in Altium Designer or open the PrjPcb file.